### PR TITLE
Deepflare disk

### PR DESCRIPTION
**Pull request by RadiantFlash7**
## About The Pull Request
This adds the long requested guns and suit that Tori sprited ages back and has been hounding litterally anyone to code in for about a month. 
To be exact, it adds: the DSS assault voidsuit, a jack of all trades, master of none-armor space suit.
The DSS Resolute: An energy carbine that can only fire in bursts of three. Has both a lethal and nonlethal function.
The DSS Stalwart: An energy scattergun, Fires a pellet child projectile modified for burn damage, and decreases in damage after 3 tiles. Functions as an in-between in our current buckshot and slug for ranges. Lethal only.
and a disk containing all three of thus, with only enough points to pick two.
image
image
image

## Why It's Good For The Game
Gives DSS a unique couple weapons, uses torisprites that they have really wanted in the game,
Also, the weapons have a unique set of use compared to our current ones, and fill a decent niche.

## Changelog
```changelog
add: DSS Stalwart
add: DSS Assault voidsuit
add: DSS Resolute
add: DSS lathe disk
```

### CODE REVIEW

File: zzzz_modular_occulus/code/modules/projectiles/guns/energy/dssguns.dm [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/obj/item/weapon/gun/energy/resolute
+	name = "DSS ECAR \"Resolute\""
+	desc = "\"DeepFlare Security System\" energy carbine. The byzantine internal capacitor design requires this gun to fire in bursts of three. It has notably become a go to fire-arm for station security forces with it's stun capabilities."
```
Comment (ShadzFen) [inline zzzz_modular_occulus/code/modules/projectiles/guns/energy/dssguns.dm:3]: It has notably become a ***go-to firearm*** for station security ***due to its*** stun capabilities.

Comment (ShadzFen) [reply zzzz_modular_occulus/code/modules/projectiles/guns/energy/dssguns.dm:3]: Other than this, the rest is fine. Just fix the description and it'll be good to go

Comment (RadiantFlash7) [reply zzzz_modular_occulus/code/modules/projectiles/guns/energy/dssguns.dm]: Fixed

File: zzzz_modular_occulus/code/game/objects/items/weapons/design_disks/dss.dm [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	name = "DSS Resolute"
+	build_path = /obj/item/weapon/gun/energy/resolute
+
+/datum/design/autolathe/gun/gemini
+	name = "DSS Gemini"
+	/obj/item/weapon/gun/energy/gun/gemini
+
+/datum/design/autolathe/gun/glasgow
+	name = "DSS Glasgow"
+	/obj/item/weapon/gun/energy/gun/glasgow
```
Comment (Michiyamenotehifunana) [inline zzzz_modular_occulus/code/game/objects/items/weapons/design_disks/dss.dm:19]: why no whitespace here

File: zzzz_modular_occulus/code/modules/projectiles/guns/energy/dssguns.dm [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+	fire_sound = 'sound/weapons/Laser.ogg'
+	rarity_value = 8
+	init_firemodes = list(
+		list(mode_name="stun", burst=3, projectile_type=/obj/item/projectile/beam/stun, fire_sound='sound/weapons/Taser.ogg', fire_delay=8, charge_cost=40, icon="stun", projectile_color = "#00FFFF"),
+		list(mode_name="kill", burst=3, projectile_type=/obj/item/projectile/beam, fire_sound='sound/weapons/Laser.ogg', fire_delay=8, charge_cost=60, icon="kill", projectile_color = "#8bbdd9"),
+		)
+
+/obj/item/weapon/gun/energy/stalwart
+	name = "DSS ESG \"Stalwart\""
+	desc = "\"DeepFlare Security System\" tri-barrel energy scattergun. The stalwart comes with a built in bayonet and fires short tripple bursts of energized particles. "
```
Comment (Michiyamenotehifunana) [inline zzzz_modular_occulus/code/modules/projectiles/guns/energy/dssguns.dm:28]: Triple with a single p.

**Review by Michiyamenotehifunana — CHANGES_REQUESTED** [review]
Basically good, just two teeny tiny issues.

File: zzzz_modular_occulus/code/modules/projectiles/guns/energy/dssguns.dm (hunk 3 above)
Comment (RadiantFlash7) [reply zzzz_modular_occulus/code/modules/projectiles/guns/energy/dssguns.dm:28]: Aaah. Thaat would be my keyboard double typing. Heck. Thanks.

File: zzzz_modular_occulus/code/game/objects/items/weapons/design_disks/dss.dm (hunk 2 above)
Comment (RadiantFlash7) [reply zzzz_modular_occulus/code/game/objects/items/weapons/design_disks/dss.dm:19]: Becaaaause... cat dumb

**Comment by RadiantFlash7** [comment]
Fixed!

**Review by Michiyamenotehifunana — APPROVED** [review]
Looks good!

**Pull request merged**